scalactic-test/src/test/scala/org/scalactic/DirectRequirementsSpec.scala

```diff
@@ -4804,7 +4804,7 @@ class DirectRequirementsSpec extends FunSpec with OptionValues {
     }
 
     it("should throw NullArgumentException with correct message when one of passed parameters through method call is null") {
-      def returnNull: String = null
+      def returnNull: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.